bootstrap-core/src/test/java/de/agilecoders/wicket/core/markup/html/bootstrap/dialog/ModalTestPanel.java

```diff
@@ -0,0 +1,30 @@
+package de.agilecoders.wicket.core.markup.html.bootstrap.dialog;
+
+import org.apache.wicket.ajax.AjaxRequestTarget;
+import org.apache.wicket.ajax.markup.html.AjaxLink;
+import org.apache.wicket.markup.html.form.Form;
+import org.apache.wicket.markup.html.form.RequiredTextField;
+import org.apache.wicket.model.Model;
+
+public class ModalTestPanel extends Modal<String>
```

Comment:
Please just add some javadoc what exactly is being tested and which issue it is related to!